docs/articles_en/documentation/openvino-ir-format/operation-sets/operation-specs/movement/strided-slice-1.rst

```diff
@@ -58,121 +121,317 @@ StridedSlice
 
 **Inputs**:
 
-*   **1**: ``data`` - input tensor to be sliced of type *T* and arbitrary shape. **Required.**
+***1**: ``data`` - input tensor to be sliced of type *T* and arbitrary shape. **Required.**
```

Comment:
```suggestion
* **1**: ``data`` - input tensor to be sliced of type *T* and arbitrary shape. **Required.**
```
+ other inputs - formatting/rendering issues in doc.

Reply:
Fixed!